File: api/v2010/short_code/update-example-1/update-example-1.5.x.php
<?php

// Update the path below to your autoload.php,
// see https://getcomposer.org/doc/01-basic-usage.md
require_once '/path/to/vendor/autoload.php';

use Twilio\Rest\Client;

// Find your Account Sid and Auth Token at twilio.com/console
$sid    = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token  = "your_auth_token";
$twilio = new Client($sid, $token);

$short_code = $twilio->shortCodes("SC6b20cb705c1e8f00210049b20b70fce3")
                     ->update(array(
                                  "smsUrl" => "http://demo.twilio.com/docs/sms.xml"
                              )
                     );

print($short_code->friendlyName);

File: api/v2010/short_code/update-example-1/update-example-1.5.x.rb
# Download the helper library from https://www.twilio.com/docs/ruby/install
require 'rubygems'
require 'twilio-ruby'

# Your Account Sid and Auth Token from twilio.com/console
# DANGER! This is insecure. See http://twil.io/secure
account_sid = 'ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX'
auth_token = 'your_auth_token'
@client = Twilio::REST::Client.new(account_sid, auth_token)

short_code = @client.short_codes('SC6b20cb705c1e8f00210049b20b70fce3')
                    .update(sms_url: 'http://demo.twilio.com/docs/sms.xml')

puts short_code.friendly_name
